src/Microsoft.AspNet.Server.Kestrel/Http/FrameHeaders.Generated.cs

```diff
@@ -7441,43 +7682,6 @@ protected override bool RemoveFast(string key)
         protected override void ClearFast()
         {
             _bits = 0;
-            
-            _CacheControl = StringValues.Empty;
-            _Connection = StringValues.Empty;
-            _Date = StringValues.Empty;
-            _KeepAlive = StringValues.Empty;
-            _Pragma = StringValues.Empty;
-            _Trailer = StringValues.Empty;
-            _TransferEncoding = StringValues.Empty;
-            _Upgrade = StringValues.Empty;
-            _Via = StringValues.Empty;
-            _Warning = StringValues.Empty;
-            _Allow = StringValues.Empty;
-            _ContentLength = StringValues.Empty;
-            _ContentType = StringValues.Empty;
-            _ContentEncoding = StringValues.Empty;
-            _ContentLanguage = StringValues.Empty;
-            _ContentLocation = StringValues.Empty;
-            _ContentMD5 = StringValues.Empty;
-            _ContentRange = StringValues.Empty;
-            _Expires = StringValues.Empty;
-            _LastModified = StringValues.Empty;
-            _AcceptRanges = StringValues.Empty;
-            _Age = StringValues.Empty;
-            _ETag = StringValues.Empty;
-            _Location = StringValues.Empty;
-            _ProxyAutheticate = StringValues.Empty;
-            _RetryAfter = StringValues.Empty;
-            _Server = StringValues.Empty;
-            _SetCookie = StringValues.Empty;
-            _Vary = StringValues.Empty;
-            _WWWAuthenticate = StringValues.Empty;
-            
-            _rawConnection = null;
-            _rawDate = null;
-            _rawTransferEncoding = null;
-            _rawContentLength = null;
-            _rawServer = null;
```

Comment:
The _raw fields are only read by `CopyToFast` which already has `if ({header.TestBit()})`, so no other changes needed here.